crates/net/network/src/config.rs

```diff
@@ -12,6 +14,19 @@ use std::{
     sync::Arc,
 };
 
+/// reexports for convenience
+#[doc(hidden)]
```

Comment:
Is this a common technique? How come you didn't pub use each thing?

Reply:
can't remember where I picked this up, but imo this is cleaner for external reexports and highlights that this should indeed be pub use